test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -190,3 +193,103 @@ var _ = SIGDescribe("[Feature:GPUDevicePlugin]", func() {
 		testNvidiaGPUs(f)
 	})
 })
+
+func testNvidiaGPUsJob(f *framework.Framework) {
+	_ = SetupNVIDIAGPUNode(f, false)
+	// Job set to have 5 completions with parallelism of 1 to ensure that it lasts long enough to experience the node recreation
+	completions := int32(5)
+	ginkgo.By("Starting GPU job")
+	StartJob(f, completions)
+
+	job, err := jobutil.GetJob(f.ClientSet, f.Namespace.Name, "cuda-add")
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
We can use `framework.ExpectNoError(err)` instead of `gomega.Expect(err).NotTo(gomega.HaveOccurred())` for readable code.
ref: https://github.com/kubernetes/kubernetes/issues/77103

Reply:
Thanks for pointing this out. Updated.